instance/models/mixins/openedx_config.py

```diff
@@ -225,8 +225,8 @@ def _get_configuration_variables(self):
             # By default, the number of workers is num_cores*4 for LMS, num_cores*2 for CMS,
             # which turns out to be too much.
             "EDXAPP_WORKERS": {
-                "lms": 3,
-                "cms": 2,
+                "lms": 14,
+                "cms": 14,
```

Comment:
Do we want CMS workers to be the same as LMS?
I thought they were usually around half that of the LMS.

Reply:
@kaizoku Well since CMS has less traffic we could probably lower it.
I'll update the PR, thanks.